templates/menu.mustache

```diff
@@ -16,9 +16,7 @@
 							title="{{getLinkTitle docObject}}">
 							{{getShortTitle docObject}}
 						</a>
-						{{#if (hasOrIsCurrent docObject)}}
-							{{> menu.mustache}}
-						{{/if}}
```

Comment:
The `hasOrIsCurrent` check prevents the child menus from being printed if present but not the current page. Upstream from this `getCurrentTree` provides the tree that is being rendered by `menu.mustache`, the generated tree does not have extra children, it generates only what needs to be printed.